cluster_management/src/main/java/com/pinterest/rocksplicator/ConfigGenerator.java

```diff
@@ -225,6 +258,10 @@ private void generateShardConfig() {
     // Write the config to ZK
     LOG.error("Generating a new shard config...");
 
+    /**
+     * TODO:
```

Comment:
Can you add your username to this and the above TODOs.

Reply:
Sure.

Reply:
Done.